### PR TITLE
US-7.5.3: Make real time notifications updates and fix new notifications order

### PR DESCRIPTION
**Pull request by SaraBoutej**
Related [commons PR](https://github.com/exodev/commons/pull/318)

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js [hunk 1]
```diff
@@ -18,4 +18,30 @@
         method: 'PATCH',
         body: operation
     });
-}
\ No newline at end of file
+}
+
+  export function getCometdSettings() {
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js]:
```suggestion
  export function getUserToken() {
```
Should be called directly by initCometd not from .vue

File: digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js [hunk 2]
```diff
@@ -18,4 +18,30 @@
...
+          } else {
+              throw new Error(('Error when getting cometd settings'));
+          }
+      })
+  }
+
+  export function initCometd(notificationSettings) {
+          const loc = window.location;
+          cCometd.configure({
+              url: `${loc.protocol}//${loc.hostname}${(loc.port && ':') || ''}${loc.port || ''}/${notificationSettings.comtedContextName}/cometd`,
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js]:
```suggestion
              url: `${loc.protocol}//${loc.hostname}${(loc.port && ':') || ''}${loc.port || ''}/cometd/cometd`,
```

File: digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js [hunk 3]
```diff
@@ -18,4 +18,30 @@
         method: 'PATCH',
         body: operation
     });
-}
\ No newline at end of file
+}
+
+  export function getCometdSettings() {
+      return fetch('/rest/notifications/cometdNotifications', {
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js]:
```suggestion
      return fetch(`/rest/continuation/gettoken/${eXo.env.portal.userName}`, {
```

File: digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js [hunk 4]
```diff
@@ -18,4 +18,30 @@
...
+          method: 'GET',
+      }).then((resp) => {
+          if(resp && resp.ok) {
+              return resp.json();
+          } else {
+              throw new Error(('Error when getting cometd settings'));
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js]:
```suggestion
              throw new Error(('Error when getting user token'));
```

File: digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js (hunk 1 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js (hunk 2 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js (hunk 3 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js (hunk 4 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js]: Done!

**Pull request merged**